### ISSUE TITLE
Citations seem out of date

### ISSUE THREAD

**Issue by simmsa** [MHKiT-Software/MHKiT#68]
The [Cite Us](https://mhkit-software.github.io/MHKiT/citeus.html) page in the documentation seems out of date. 

* Authors:
  * Does it make sense to update the authors to the current contributors of each package?
* Abstract Note:
  * References to `Code Hub` should be changed to `Software`
* Year
  * How should this be handled? Technically 2020 was the year of inception, but the software has changed significantly between 2020 and the present.

**Comment by akeeste** [comment]
I agree, the current citation is pretty dated. We should check what current best practices are for software citation. A few routes we can go based on other projects I work on:
- Create a new DOI for each release through Zenodo. Ask users to cite the DOI of the version of MHKiT that they use. This gives an very long list of contributors in order of GitHub stats (commits?). There's no other information tied to the DOI, just the software release.
- Change the citation to a new publication (conference paper this summer or JOSS would be a relatively low lift) with the current development team

**Comment by ssolson** [comment]
A new citation is generated for every release. The cite all versions is the one we should show. This list of Authors should be updated with the github contributors. There is a "cite all version" Option  https://zenodo.org/records/11148583

Basically no one uses this so I wouldn't spend too much brain power on it.

**Comment by rebeccamccabe** [comment]
hi, I am a user of MHKit and I got an error when using the provided bibtex key for matlab. I wanted to let you know that there is a missing bracket after the url, bolded below.
<pre><code>
@misc{MHKiT_MATLAB,
    title = {MHKiT (Marine and Hydrokinetic Toolkit) - MATLAB},
    author = {Pauly, Rebecca and Klise, Katherine and Ruehl, Kelley M. and Olson, Sterling and Shippert, Timothy and Morrell, Zachary and Bredin, Sarah and Lansing, Carina and Macduff, Matt and Martin, Tonya and Sivaraman, Chitra and Gunawan, Budi and Driscoll, Frederick},
    abstractNote = {The MHKiT Code Hub is a collection of open-source marine renewable energy (MRE) software. The MHKiT code, developed in Python and MATLAB, includes modules for ingesting, quality controlling, processing, visualizing, and managing data. MHKiT-Python and MHKiT-MATLAB provide robust and verified functions in both Python and MATLAB that are needed by the MRE community to standardize data processing. Calculations and visualizations adhere to IEC technical specifications and other guidelines. Current functionality includes power performance, power quality, mechanical loads, resource tools, and data quality control for wave, tidal, and river applications.},
    url = {https://doi.org/10.5281/zenodo.3928405<b>}</b>,
    howpublished = {[Computer Software] \url{https://doi.org/10.5281/zenodo.3928405}},
    year = {2020},
    month = {jan},
    doi       = {10.5281/zenodo.3928405}
}
